server/routes/findInterventions/searchSummaryPresenter.test.ts

```diff
@@ -0,0 +1,58 @@
+import InterventionsFilter from './interventionsFilter'
+import SearchSummaryPresenter from './searchSummaryPresenter'
+import pccRegionFactory from '../../../testutils/factories/pccRegion'
+import { PCCRegion } from '../../services/interventionsService'
+
+describe(SearchSummaryPresenter, () => {
+  describe('summary', () => {
+    describe('with an empty filter', () => {
+      it('returns no summary list items', () => {
+        const presenter = new SearchSummaryPresenter(new InterventionsFilter(), [])
+
+        expect(presenter.summary).toEqual([])
+      })
+    })
+
+    function linesForKey(key: string, args: [InterventionsFilter, PCCRegion[]]): string[] {
```

Comment:
I know the args types are slightly different here, but might be worth extracting the `linesForKey` function in `interventionDetailsPresenter` for this to some kind of "test helper" as this is very similar (on a quick glance, I could be wrong though...). Again, we can do this later, but if so, let's add a card for it.

Reply:
Good shout - [done](https://github.com/ministryofjustice/hmpps-interventions-ui/pull/128/commits/91b6079c96d90732621d3f52c3f6a075ab550798).